src/pages/api/taskQueueHandler.ts

```diff
@@ -0,0 +1,65 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { Twilio } from 'twilio'
+import {
+  TaskQueueInstance,
+  TaskQueueListInstanceCreateOptions,
+  TaskQueueContextUpdateOptions,
+} from 'twilio/lib/rest/taskrouter/v1/workspace/taskQueue'
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  const { method } = req
+  const { taskQueueSid } = req.query as {
+    taskQueueSid: string
+  }
+  const workspaceSid = process.env.TWILIO_TASKROUTER_WORKSPACE_SID;
```

Comment:
My understanding is that different orgs under ASA will have different workspaces, so we can't programmatically store all their workspaceSids as environment variables. Additionally, IDs are generally safe to expose publicly. This is why the original code passes workspaceSid from the frontend to the backend.

Reply:
Per the discussion w/ Quan and Chris today each org will have their own subaccount. For now, use taskQueue.ts and pass in the twilio workspace sid using `process.env`. If you have questions don't hesitate to reach out to either me or Quan!